Fix hang in Corrections tab (#41228)
mantidproject/mantid: docs/source/release/v6.16.0/Inelastic/Bugfixes/41147.rst

> - :ref:`Container Subtraction <container-subtraction>` interface does not delete the subtracted workspace from the ADS when the windows is closed.

Reviewer: ```suggestion
- The :ref:`Container Subtraction <container-subtraction>` interface no longer deletes the subtracted workspace from the ADS when the window is closed.
```

> - The :ref:`Corrections <interface-inelastic-corrections>` interface does not hang Mantid when closing the interface after using a container subtracted workspace in the :ref:`Apply Absorbtion corrections <apply_absorp_correct>` tab.

Reviewer: ```suggestion
- The :ref:`Corrections <interface-inelastic-corrections>` interface no longer hangs on close after using a container-subtracted workspace in the :ref:`Apply Absorbtion corrections <apply_absorp_correct>` tab.
```

> - :ref:`Container Subtraction <container-subtraction>` interface has now an output name widget for adding a label to outputs on the ADS.

Reviewer: ```suggestion
- The :ref:`Container Subtraction <container-subtraction>` interface now includes an output-name widget for labelling outputs in the ADS.
```
